px-datetime-range-behavior.html

```diff
@@ -70,53 +70,12 @@
      */
     presetRanges: Object, //may be useless
 
-    /**
-     * (required: 'px-{datetime}-range-{picker}')
-     *
-     * This is the object you use to instantiate px-datetime components. You can retrieve the object through 2-way data-binding.
-     *
-     * Formated as ISO8601 strings
-     * ```
-     * {
-     *    "from": "2013-01-07T22:44:30.652Z",
-     *    "to" : "2013-02-04T22:44:30.652Z"
-     * }
-     * ```
-     *
-     * This object which will be changed when the user Submits their new selected range.
-     *
-     * @default the last 7 days
-     */
-    range: {
```

Comment:
wondering if we should provide a "one way" of listening to changes for range now that `range` is gone. Maybe an event when either `fromMoment` or `toMoment` changes?

Reply:
Our thought is in `px-datetime-range-field` after validation we would fire an event saying the Moments changed. This way someone could listen to that event for when either one of the momentObjs updated. Or they could listen to `from-moment-changed` and  `to-moment-changed` to know which one was updated.